docs/README.kernel.md

```diff
@@ -0,0 +1,120 @@
+# ﻿NI Linux Real-Time Kernel Source README
+
+## Introduction
+
+Use these instructions to build a kernel that can boot on NI Linux Real-Time
+(NILRT) targets.
+
+Note: National Instruments does not support kernels other than one provided by
+National Instruments. Any kernel other than what is provided by National
+Instruments may not have the same performance, determinism, features, or basic
+functionality.
+
+
+### Requirements
+
+1. A linux build machine
+2. GNU `make`
+3. The `mksquashfs` utility
+
+
+## Downloading the Source
+
+The kernel source for NI Linux Real-Time kernel is available for download from
+NI's fork of the linux repo. The repo follows a mainline-branching model with
+mainline branch refs like `nilrt/master/${linux_version_base}` and product release branch refs like `nilrt/${product_version}/${linux_version_base}`.
+
+```bash
+git clone https://github.com/ni/linux.git
+cd linux
+git checkout nilrt/20.6/4.14  # the NI 20.6 release kernel
+```
+
+
+### Using the Tools
+
+You can use any cross-compilation toolchain of your choosing, or you can
+download the toolchains from ni.com
+([armv7](https://www.ni.com/en-us/support/downloads/software-products/download.gnu-c---c---compile-tools-for-armv7.html#338449),
+[x64](https://www.ni.com/en-us/support/downloads/software-products/download.gnu-c---c---compile-tools-x64.html#338443)),
+or build them yourself.
+
+Refer to the README to get started with building OpenEmbedded components, and
+build the NILRT SDK containing a GCC toolchain.
+
+The README describes how to build SDKs for both Linux and Windows machines.
```

Comment:
Which part of the README describes this?

Reply:
[The part that I removed](https://github.com/ni/nilrt/pull/36/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L127-L165) because I don't think it works anymore and, if it does, I don't know how useful it is anymore.

I intentionally didn't bother changing the content of the kernel README until I hear @gratian 's opinion about how far we are from it being accurate.